bin/test-boot

```diff
@@ -56,6 +56,7 @@ set -x  # show next command
 
 qemu-system-$GUESTARCH -m $MEM -machine $MACHINE \
    -device virtio-net-pci,netdev=net00 \
+   -device virtio-rng-pci \
```

Comment:
In other places I've invoked this as:

 
     -device virtio-rng-pci,rng=rngid0,id=rng0
     -object rng-random,filename=/dev/urandom,id=rng0

the key thing being that you tie the virtio-rng-pci to host's /dev/urandom rather than the default /dev/random.
At least for our tests, /dev/urandom is "good enough".